### PR TITLE
[ADD] allow to pass force_deps to migrate.py

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by legalsylvain** [comment]
:+1: thanks.

File: scripts/migrate.py [hunk 1]
```diff
@@ -167,6 +167,10 @@ def copy_database(conn_parms):
 parser.add_option("-I", "--inplace", action="store_true", dest="inplace",
                   help="don't copy database before attempting upgrade "
                   "(dangerous)")
+parser.add_option(
+    "-F", "--force-deps", action="store", dest="force_deps",
+    help="force dependencies from a dict of the form \"{'module_name': "
+    "['new_dependency1', 'new_dependency2']}\"")
```
Comment (pedrobaeza) [inline scripts/migrate.py]: Should you check if the parameter is a well-formed dict?

**Comment by legalsylvain** [comment]
@hbrunn : any news about @pedrobaeza's remark ?

**Comment by hbrunn** [comment]
@legalsylvain @pedrobaeza thanks for the reminder, I seem to have overlooked this one. Commited just now

**Comment by legalsylvain** [comment]
:+1: Thanks.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**